package/MDAnalysis/analysis/base.py

```diff
@@ -22,17 +21,19 @@
 classes.
 
 """
-from six.moves import range
-
+import inspect
```

Comment:
Compatibility imports should come first as they can change the language behavior.